cfi_backward.adoc

```diff
@@ -192,9 +204,13 @@ The `sspopchk` instruction and its compressed form `c.sspopchk` can be used to
 pop the shadow return address value from the shadow stack and check that the
 value matches the contents of the link register.
 
-The `sspop` instruction can be used to pop a return address from the shadow
+The `ssload` instruction can be used to load a return address from the shadow
 stack into a link register.
 
+The `sspinc` instruction adds the zero-extended  non-zero immediate `nzuimm`,
```

Comment:
What happens with the `nzuimm == 0` encoding? Trap?

Reply:
Zimop codepoints that are not redefined by this extension follow their Zimop defined behavior.

Reply:
OK, in compressed instructions where I see existing `nzuimm` field instances, the 0 encoding is called "reserved" so I wanted to check. It's not obvious to me that your description here leaves the 0 encoding as a "hole" to be caught by the underlying Zimop.

Reply:
The specification states:
"When a Zimop encoding is not used by the Zisslpcfi extension then the instruction follows its Zimop defined behavior."
I see perhaps a statement missing that the code point with `nzuimm` of 0 is not  defined by  Zisslpcfi. I will add that,

Reply:
Whatever you think. For reference, when I searched `nzuimm` in my old copy of the ISA, I hit this: "The code points with nzimm=0 are reserved." so I wondered if that might apply here.

Reply:
(that's in the section on `C.LUI`)

Reply:
Added a statement "Only non-zero encodings of `nzuimm` are defined for `sspinc`."